.github/ISSUE_TEMPLATE/OTHER.yml

```
@@ -0,0 +1,12 @@
+name: "Other Report 🌐"
+description: Report an issue or possible bug.
```

```suggestion
description: Report something else we should know about the docs site!
```
Since we already have a category for bugs, I feel like this description should leave it way more open.